CompatBot/Commands/Misc.cs

```diff
@@ -18,29 +18,32 @@ internal sealed class Misc: BaseCommandModuleCustom
 
         private static readonly List<string> EightBallAnswers = new List<string>
         {
-            // 37
+            // 43
             "Ya fo sho", "Fo shizzle mah nizzle", "Yuuuup", "Da", "Affirmative", // 5
             "Sure", "Yeah, why not", "Most likely", "Sim", "Oui",
             "Heck yeah!", "Roger that", "Aye!", "Yes without a doubt m8!", "<:cell_ok_hand:324618647857397760>",
             "Don't be an idiot. YES.", "Mhm!", "Many Yes", "Yiss", "Sir, yes, Sir!", 
             "Yah!", "Ja", "Umu!", "Make it so", "Sure thing",
             "Certainly", "Of course", "Definitely", "Indeed", "Much yes",
             "Consider it done", "Totally", "You bet", "Yup", "Yep",
-            "Certainly", "Yarp",
+            "Certainly", "Yarp", "Very well", "Will do", "Aye mate",
```

Comment:
“Certainly” is a duplicate, “very well” and “will do” do not work very well as an answer (“should I wash the dishes?” “Will do” “um what”)